src/main.cpp

```diff
@@ -1,34 +1,47 @@
 #include <iostream>
 #include <serial/serial.h>
 #include <fmt/format.h>
+
 int main(int argc, char** argv)
 try
 {
-    constexpr auto port = "COM20";
+	constexpr auto port = "COM1";
+
+	std::cout << "Using Port: ";
+	std::cout << port << std::endl;
+	auto serialPort = serial::Serial(port, 115200);
+	if(!serialPort.isOpen()) 
+	{ 
+		throw std::runtime_error(fmt::format("Port {} not open", port));
+	}
+	serialPort.write("Hello, World!\n");
+	std::string message {};
+	serialPort.readline(message);
```

Comment:
please configure your ide to use the clang-format file to make indentations homogeneous